dace/codegen/compiled_sdfg.py

```diff
@@ -5,6 +5,7 @@
 import re
 import shutil
 import subprocess
+from subprocess import PIPE
```

Comment:
Please omit the extra import and just use `subprocess.PIPE` in the code

Reply:
I agree as well